### ISSUE TITLE
Editor folds: Unintuitive cut-and-paste

### ISSUE THREAD

**Issue by TheBizzle** [NetLogo/NetLogo#2316]
### Code to reproduce

```netlogo
to abc
  let grapes 3
end
```

### Steps to reproduce

  * Fold up `abc`
  * Select `to abc` (not the lines after it)
  * Cut

### What should happen

It cuts the entire procedure.

### What actually happens

It cuts only `to abc`.

**Comment by ERSUCC** [comment]
I disagree that this is not how it should behave, because someone might want to copy just the procedure declaration without its body. If you want to select the entire folded procedure, you can select from the beginning of the declaration to the beginning of the following line, and it will copy the procedure body as well.

**Comment by TheBizzle** [comment]
But why do I have the procedure folded up if I don't want to treat the entire thing as one unit?  Also: when the fold is the last line of code in the model, there is no ability to select to the following line.

**Comment by ERSUCC** [comment]
That's a good point, I suppose it should be changed.
